wavefront/server/modules/plugins_module/plugins_module/controllers/__init__.py

```
@@ -1,4 +1,5 @@
 from .datasource_controller import datasource_router
 from .authenticator_controller import authenticator_router
+from .cloud_storage_controller import cloud_storage_router
 
-__all__ = ['datasource_router', 'authenticator_router']
+__all__ = ['datasource_router', 'authenticator_router', 'cloud_storage_router']
```

Ruff (0.15.2)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)